moveit_ros/moveit_servo/src/servo_parameters.cpp

```diff
@@ -320,6 +327,19 @@ ServoParameters ServoParameters::get(const std::string& ns,
       node_parameters->get_parameter(ns + ".hard_stop_singularity_threshold").as_double();
   parameters.joint_limit_margin = node_parameters->get_parameter(ns + ".joint_limit_margin").as_double();
 
+  if (node_parameters->has_parameter(ns + ".leaving_singularity_threshold_multiplier"))
+  {
+    parameters.leaving_singularity_threshold_multiplier =
+        node_parameters->get_parameter(ns + ".leaving_singularity_threshold_multiplier").as_double();
+  }
+  else
+  {
+    parameters.leaving_singularity_threshold_multiplier = 1.0;
+    RCLCPP_WARN(LOGGER, "Using default servo singularity handling; add parameter "
+                        "'leaving_singularity_threshold_multiplier' to define leaving singularity threshold. See "
+                        "https://github.com/ros-planning/moveit2/pull/620 for more information.");
```

Comment:
Should this warning have the word `deprecated` in it to let users know that we might remove this behavior in the future?